site/constants/index.ts

```diff
@@ -31,10 +31,23 @@ export const DISRUPTION_REASONS: {
     },
 ];
 
+export const DISRUPTION_TYPES = ["planned", "unplanned"];
+
+export const CD_DATE_FORMAT = "DD/MM/YYYY";
 export const COOKIES_POLICY_COOKIE = "cdd-cookies-policy";
 
 export const COOKIE_PREFERENCES_COOKIE = "cdd-cookie-preferences-set";
 
 export const oneYearInSeconds = 31556952;
 
+export const TEN_SECONDS_IN_MILLISECONDS = 10000;
```

Comment:
Is this needed as a constant?

Reply:
Added to avoid using the default cookie lifetime value. This value is currently used to set lifetime of multiple cookies.

Reply:
Another example would be if we used the number 20 in a lot of different files, it wouldnt make sense to have:

`const twenty = 20;`

in a constants file, because we can just write 20.